tests/e2e/support/utils.js

```diff
@@ -282,6 +291,8 @@ export function testNumberOfVertices(expectedVertices) {
         return;
       }
 
+      /* Uncomment the following line when the new download XML button will be re-added
```

Comment:
As the download button will be present in the process list, remove this commented code